platform/windows/crash_handler_windows.h

```diff
@@ -35,12 +35,15 @@
 #include <windows.h>
 
 // Crash handler exception only enabled with MSVC
-#if defined(DEBUG_ENABLED) && defined(MSVC)
+#if defined(DEBUG_ENABLED)
 #define CRASH_HANDLER_EXCEPTION 1
 
+#ifdef MSVC
```

Comment:
I noticed this while working on another PR - is `MSVC` actually a valid define? Usually this is `_MSC_VER`.

Reply:
It's set in the `detect.py` https://github.com/godotengine/godot/blob/e413d1c9c35bec80f7244772bd63a3269f56c5a2/platform/windows/detect.py#L371, so it's valid but `_MSC_VER` is probably better.

Reply:
Ah yeah I'd remove that one from `detect.py`. We're only using it in a couple of files:
```
$ rg "ifn?def MSVC"
drivers/unix/file_access_unix.cpp
47:#ifdef MSVC

$ rg "defined\(MSVC\)"
platform/windows/crash_handler_windows.h
38:#if defined(DEBUG_ENABLED) && defined(MSVC)
```

Reply:
Rebased and replaced it with `_MSC_VER` everywhere.